app/views/layouts/exercise_inputs/layouts/_input_kids.html.erb

```diff
@@ -20,12 +20,13 @@
 
         <div class="mu-kids-states">
           <div class="mu-kids-state mu-state-initial mu-kids-gbs-board-initial">
-            <strong>Tablero inicial</strong>
+            <strong id="initialBoardText"><%= t :initial_board %></strong>
+            <strong id="actualBoardText"><%= t :actual_board %></strong>
             <div class="mu-kids-state-image"><%= @exercise.initial_state&.html_safe %></div>
```

Comment:
I agree that this is better than the hardcoded spanish wording. But we are still coupling the translation to gobstones. 

I also understand that decoupling it doesn't worth the effort yet - so I am not asking to change the actual translation yet -, but could we at least name the translations in a consistent manner with the model? They should be `actual_state` and `initial_state`.

Reply:
I agree it's inconsistent, but i didn't like the idea of having a translation like: "initial_state: Initial board"